docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md

```diff
@@ -381,7 +381,7 @@ The **Work Area**: is displaying **.../{% include product %}/projects/the_other_
 
 ### Extending the example
 
-In this example, we modified a single template, but there’s plenty more you can do with your filesystem configuration. In a real world example, you’d likely change *all* asset-related files to have the same file naming conventions. You can make modifications based on other entities (Season, Episode, Level, etc.), create user folders, name your folders based on {% include product %} data manipulated with regular expressions, and much more. You can learn about all of Toolkit’s folder and schema options in the [Filesystem Configuration Reference](https://support.shotgunsoftware.com/hc/en-us/articles/219039868). 
```

Comment:
Another couple of curly quotes.  I think maybe we should just do a separate PR to find and replace all of these, if they do indeed cause any kind of publishing issue (which it doesn't look like they do).

Reply:
I see {% include product %}, but I'm not seeing any additional curly quotes... can you let me know if that's right?

Reply:
Ah! Sorry - my brain is still coming online... curly quotes, not %}! Yes, I feel like this should be another PR to clean these up as well.